components/common/mojomUtils.ts

```diff
@@ -32,3 +38,8 @@ export function mojoTimeToJSDate (mojoTime: mojo.Time): Date {
 export function mojoTimeDeltaToJSDate (mojoTime: mojo.TimeDelta) {
   return new Date(Number(mojoTime.microseconds) / 1000)
 }
+
+export function getKeysForMojomEnum(mojomEnum: MojomEnum): string[] {
```

Comment:
Optionally, we could return the actual type strings of the enum:

```ts
export function getKeysForMojomEnum<T extends MojomEnum>(e: T): (Exclude<keyof T, 'MIN_VALUE' | 'MAX_VALUE'>)[] {
    const ignoreKeys = ['MIN_VALUE', 'MAX_VALUE']
    return Object.keys(e).filter(k => !ignoreKeys.includes(k)) as any
}
```

Reply:
ooh that's nice